### ISSUE TITLE
Issue with connecting WebSocket on startup

### ISSUE THREAD

**Issue by VladShaternik** [webfolderio/cdp4j#137]
Sometimes when the Launcher is launched CdpException is thrown with the message "WebSocket connection url is required!". It is being thrown from the Launcher.launchWithProcessBuilder(List<String> arguments) method.

I think the problem is in the following:
The process is getting started, then immediately after that, the Scanner for the ErrorStream of the process is created. There is a small chance that the process did not write to the error stream anything yet, but the Scanner is immediately checking for the content (by calling scanner.hasNext()) in the while loop. Therefore, while loop is immediately skipped and the WebSocketConnection is never established.

![image](https://user-images.githubusercontent.com/37490669/93039192-b75ad080-f5fb-11ea-8292-1758ff511e1f.png)

There needs to be some other way for checking ErrorStream for "devtools listening on". It will be nice if it will be checking for "devtools listening on" until some timeout and only after that timeout throw a CdpException.

**Comment by webfolderio** [comment]
Yes, thanks for the detailed bug report. This error error mostly occured if more than one Chrome process share the same user data dir (user profile directory). We couldn't find a proper way to solve this issue.

**Comment by VladShaternik** [comment]
So if this error happens scanner never contains "devtools listening on"?
Do you think checking for "devtools listening on" until a specified timeout would solve a problem? Maybe there is just some delay before it is sent to the error stream.
